test/unit/org/apache/cassandra/tools/cassandrastress/CassandrastressTest.java

```diff
@@ -70,6 +70,8 @@ public void testNodesArg()
 
         String hostNameAndPort = String.format("localhost:%s", nativePort);
         invokeAndAssertCleanExit(baseArgs, "-node", hostNameAndPort);
+
+        invokeAndAssertCleanExit(baseArgs, "-mode", "simplenative", "prepared");
```

Comment:
@dmi0 could you add here more combinations? like

-mode simplenative prepared
-mode simplenative unprepared
-mode prepared
-mode unprepared

This should be applied to all branches.

Reply:
@smiklosovic  done